test/Interop/Cxx/enum/anonymous-with-swift-name.swift

```diff
@@ -0,0 +1,34 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-experimental-cxx-interop)
+
+// REQUIRES: executable_test
+
+import AnonymousWithSwiftName
+import StdlibUnittest
+
+var AnonymousEnumsTestSuite = TestSuite("Anonymous Enums With Swift Name")
+
+AnonymousEnumsTestSuite.test("SOME_OPTIONS") {
+  let red: SOColorMask = .red
+  let green = SOColorMask.green
+  let blue = .blue as SOColorMask
+  let all: SOColorMask = .all
+
+  expectEqual(red.rawValue, 2)
+  expectEqual(green.rawValue, 4)
+  expectEqual(blue.rawValue, 8)
+   expectEqual(all.rawValue, ~CUnsignedInt(0))
+}
+
+AnonymousEnumsTestSuite.test("CF_OPTIONS") {
+  let red: SOColorMask = .red
+  let green = SOColorMask.green
+  let blue = .blue as SOColorMask
+  let all: SOColorMask = .all
```

Comment:
These need to be CF

Reply:
Fixed in: https://github.com/apple/swift/pull/42431/commits/1d747131a2d2cf6f5a4d7e077e0ac0ff7ffedf48